packages/MainUI/components/types.ts

```diff
@@ -0,0 +1,46 @@
+import { SxProps } from '@mui/material';
+import { Tab } from '@workspaceui/etendohookbinder/src/api/types';
+
+export const IconSize = 16;
+
+export interface ToolbarProps {
+  windowId: string;
+  tabId?: string;
+}
+
+export type IconName = 'plus' | 'save' | 'trash' | 'refresh-cw' | 'search' | 'grid' | 'download' | 'paperclip';
+
+export interface ButtonResponse {
```

Comment:
This one looks exactly like the declared in etendo hook binder. Reuse that one and remove this